src/test/a11y/test-suite.js

```diff
@@ -22,6 +22,7 @@ const CARD_ADDRESS_URL = `${BASE_URL}/card-address`
 const PHONE_NUMBER_URL = `${BASE_URL}/phone-number`
 const EMAIL_ADDRESS_URL = `${BASE_URL}/email-address`
 const CHECK_URL = `${BASE_URL}/check`
+const CHOOSE_CHANNEL_FOR_CODE_URL = `${BASE_URL}/choose-channel-for-code`
```

Comment:
Just checking this is the URL specified by UX?

Reply:
That was choose-channel but Matt and I thought choose-channel-for-code would work better.

Reply:
Something to double-check with Sam tomorrow...

Reply:
My thoughts were that `channel` is a strange word to be using, but if they're happy with it, then I guess it's fine. My first thought would be to use the word `method` instead of `channel`.

Reply:
or 'choose-where-we-send-code'...